core/scripts/handle.py

```diff
@@ -0,0 +1,108 @@
+"""
+Handle handles creating and updating WCosa projects
+"""
+
+from shutil import copyfile
+from colorama import Fore
```

Comment:
Seems like our only dependency so far. We should make a `setup.py` to deal with these things and to create a proper python project

Reply:
yeah that is the plan eventually